util/src/main/java/org/datacommons/util/PlaceSeriesSummary.java

```diff
@@ -17,15 +17,21 @@
 
 // An object to hold all the series information for all the statistical variables for a place.
 public class PlaceSeriesSummary {
+
   public static class SeriesSummary {
     StatValidationResult.Builder validationResult;
+    // Key is date of each datapoint. Use treemap here to keep the dates sorted.
     TreeMap<String, DataPoint> timeSeries;
   }
 
+  public static boolean TEST_mode = false;
+  // Key in svSeriesSummaryMap is stat var dcid & key in the Map<Long, SeriesSummary> is a hash
+  // constructed using place dcid, stat var dcid, measurement method, observation period,
```

Comment:
nit: do we need stat-var dcid again?

Reply:
we don't need the stat-var dcid again, but the string used to make the hash is generated using the StatValidationResult which contains the stat-var dcid. Since including the stat var dcid doesn't require any additional memory (the hash is just a Long created from a string), I don't think having the stat var dcid will cause any problems